cmd/kpod/import.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/projectatomic/libpod/libpod"
+
+	"github.com/opencontainers/image-spec/specs-go/v1"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	importFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "change, c",
+			Usage: "Apply imgspecv1 configurations to the created image",
+		},
+		cli.StringFlag{
+			Name:  "message, m",
+			Usage: "Set commit message for the image imported",
+		},
+	}
+	importDescription = `Imports a tarball and saves it as a root filesystem image.
```

Comment:
Create a container image from the contents of the specified tarball  (.tar, .tar.gz, .tgz, .bzip, .tar.xz, .txz).  
Note remote tar balls can be specified, via web address.

Optionally tag the image.  You can specify the Dockerfile instructions using the --change option.

Reply:
fixed